src/test/java/api/support/fixtures/LoansFixture.java

```diff
@@ -164,6 +166,16 @@ public Response attemptClaimItemReturned(ClaimItemReturnedRequestBuilder request
       claimItemReturnedURL(request.getLoanId()), "attempt-claim-item-returned-request");
   }
 
+  public Response resolveClaimAsMissing(ResolveClaimAsMissingRequestBuilder request) {
+    return restAssuredClient.post(request.create(),
+      resolveClaimAsMissingUrl(request.getLoanId()), 204, "resolve-claim-as-missing-request");
+}
+
+  public Response attemptResolveClaimAsMissing(ResolveClaimAsMissingRequestBuilder request) {
+    return restAssuredClient.post(request.create(),
+      resolveClaimAsMissingUrl(request.getLoanId()), "attempt-resolve-claim-as-missing-request");
+  }
+
```

Comment:
This class is getting pretty large, I might be tempted to extract the claimed returned related parts of it out into a separate fixture.

Reply:
Moved to `api.support.fixtures.ClaimItemReturnedFixture`